exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -270,71 +387,71 @@ func (e *Exchange) UpdateTicker(ctx context.Context, currencyPair currency.Pair,
 	if err := e.UpdateTickers(ctx, a); err != nil {
 		return nil, err
 	}
-	return ticker.GetTicker(e.Name, currencyPair, a)
+	pFormat, err := e.GetPairFormat(a, false)
```

Comment:
🍻 For efficiency we should be making this a no-op return an common.ErrNotSupported and wrap batch is available. Unless there is a specific endpoint for 1 item fetching. A client can get the cached ticker.